config/locales/emails/provider_mailer.yml

```diff
@@ -1,5 +1,10 @@
 en:
   provider_mailer:
+    subject: "%{subject} - manage teacher training applications"
+    account_created:
+      subject: Sign in
```

Comment:
just to double check the sign in emails were specified to rely on the appended text and removing `to Manage teacher training applications`?

Reply:
Yes!